Packs/Troubleshoot/IncidentFields/incidentfield-Instance_Name.json

```diff
@@ -0,0 +1,34 @@
+{
+ "associatedToAll": true,
+ "caseInsensitive": true,
+ "cliName": "instancename",
+ "closeForm": false,
+ "content": true,
+ "description": "Name of the instance to troubleshoot",
+ "editForm": true,
+ "fieldCalcScript": "096f78c3-e8a5-42ee-82e5-1540fc285d3f",
+ "group": 0,
+ "hidden": false,
+ "id": "incident_instancename",
+ "isReadOnly": false,
+ "locked": false,
+ "name": "Instance Name",
+ "neverSetAsRequired": false,
+ "ownerOnly": false,
+ "propagationLabels": [
```

Comment:
lets run format on the fields, i dont think `propagationLabels` should be here

Reply:
its after format

Reply:
@roysagi is this intentional ?

Reply:
I see propagation labels are set in the schema
![image](https://user-images.githubusercontent.com/50295826/104127103-e21a2300-5368-11eb-9a21-b2aaea12617d.png)
They were added by @barchen1 a month ago - can you give us any insight as to why?

Reply:
`propagationLabels` is used to sync "specific" content items based on labels to tenants https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-multi-tenant-guide/manage-content/manage-content-overview.
we probably don't need it as defaults in our content cause its more of a customer customization option